app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -27,8 +27,9 @@ class MainFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-        // Synthetics aren't working at the moment and neither is data binding
-        // One of our UI designers needs to get on this soon - Aidan
+        /* Synthetics aren't working at the moment and neither is data binding
+          /* One of our UI designers needs to get on this soon - Aidan */
+         */
```

Comment:
Changes this to a comment block with a nested comment, per kotlin standards. https://kotlinlang.org/docs/basic-syntax.html#comments

Reply:
Instead of changing the comment, I'd focus on fixing the error that the comment is describing.  :)